src/main/scala/is/hail/io/bgen/BgenRecord.scala

```diff
@@ -99,7 +98,7 @@ final class Bgen12GenotypeIterator(a: Array[Byte],
 
   private val sampleProbs = ArrayUInt(nGenotypes)
 
-  private val pa = new BGen12ProbabilityArray(a, nSamples, nGenotypes, nBitsPerProb)
+  private val pa = new Bgen12ProbabilityArray(a, nSamples, nGenotypes, nBitsPerProb)
```

Comment:
I would be OK with BGEN throughout.  Vcf and the look wrong to me visually.

Reply:
I want to save all of the renaming of Bgen to BGEN for a future PR.